corehq/form_processor/tests/test_extension_cases.py

```diff
@@ -94,6 +94,34 @@ def test_extension_of_multiple_hosts_returned(self):
         returned_cases = CommCareCaseIndex.objects.get_extension_case_ids(self.domain, [host_id])
         self.assertItemsEqual(returned_cases, [extension_id])
 
+    def test_closing_any_host_closes_extension(self):
+        host_1_id = uuid.uuid4().hex
+        host_1_struct = CaseStructure(case_id=host_1_id, attrs={'create': True})
+        host_2_id = uuid.uuid4().hex
+        host_2_struct = CaseStructure(case_id=host_2_id, attrs={'create': True})
+        extension_id = uuid.uuid4().hex
+        self.factory.create_or_update_case(
+            CaseStructure(
+                case_id=extension_id,
+                indices=[
+                    CaseIndex(
+                        host_1_struct,
+                        relationship=CASE_INDEX_EXTENSION,
+                        identifier="host_1",
+                    ),
+                    CaseIndex(
+                        host_2_struct,
+                        relationship=CASE_INDEX_EXTENSION,
+                        identifier="host_2",
+                    ),
+                ],
+                attrs={'create': True},
+            )
+        )
+        self.factory.close_case(host_1_id)
+        extension = CommCareCase.objects.get_case(host_1_id, domain=self.domain)
```

Comment:
Should `host_1_id` not be `extension_id`

Reply:
Well spotted! Thank you!